doc/en/user/source/configuration/status.rst

```diff
@@ -23,51 +23,69 @@ The following table describes the current status indicators.
+       
    * - JAI Maximum Memory
-     - Expresses in bytes the amount of memory available for tile cache, in this case 455 Mbytes.
+     - The amount of memory available for image processing tile cache, in this case 1.78 GB.
    * - JAI Memory Usage
-     - Run-time amount of memory is used for the tile cache. Clicking on the "Free Memory" button, clears available JAI memory by running the tile cache flushing.
+     - Run-time amount of memory is used for the tile cache.
+       
+       Press :guilabel:`Free Memory`" to clear available JAI memory by running the tile cache flushing.
    * - JAI Memory Threshold
      - Refers to the percentage, e.g. 75, of cache memory to retain during tile removal.
    * - Number of JAI Tile Threads
      - The number of parallel threads used by the scheduler to handle tiles.
    * - JAI Tile Thread Priority
      - Schedules the global tile scheduler priority. The priority value defaults to 5, and must fall between 1 and 10.
-   * - ThreadPoolExecutor Core Pool Size
+   * - Thread Pool Core Pool Size
      - Number of threads that the ThreadPoolExecutor will create. This is underlying Java runtime functionality - see the Java documentation for ThreadPoolExecutor for more information.
-   * - ThreadPoolExecutor Max Pool Size
+   * - Thread Pool Max Pool Size
      - Maximum number of threads that the ThreadPoolExecutor will create. This is underlying Java runtime functionality - see the Java documentation for ThreadPoolExecutor for more information.
-   * - ThreadPoolExecutor Keep Alive Time (ms)
+   * - Thread Pool Keep Alive Time
      - Timeout for threads to be terminated if they are idle and more than the core pool number exist. This is underlying Java runtime functionality - see the Java documentation for ThreadPoolExecutor for more information.
    * - Update Sequence
-     - Refers to the number of times (426) the server configuration has been modified.
+     - Refers to the number of times, e.g. 157, the server configuration has been modified.
    * - Resource cache
-     - GeoServer does not cache data, but it does cache connection to stores, feature type definitions, external graphics, font definitions and CRS definitions as well. The "Clear" button forces those caches empty and makes GeoServer reopen the stores and re-read image and font information, as well as the custom CRS definitions stored in `${GEOSERVER_DATA_DIR}/user_projections/epsg.properties`.
+     - GeoServer does not cache data, but it does cache connection to stores, feature type definitions, external graphics, font definitions and CRS definitions as well. Including custom CRS definitions defined in data directory :file:`user_projections/epsg.properties`.
+     
+       Press :guilabel:`Clear` to force these caches empty and makes GeoServer reopen the stores and re-read image and font information, as well as the custom CRS definitions. 
```

Comment:
```suggestion
       Press :guilabel:`Clear` to empty the resource cache. This will force GeoServer to reconnect to stores, reconnect to databases, re-read icon and font information, and reload custom CRS definitions. 
```